packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -11116,6 +11116,30 @@ components:
         type: array
         items:
           $ref: "#/components/schemas/EServiceDescriptorAttribute"
+    EServiceDescriptorCertifiedAttributesResponse:
+      type: object
+      additionalProperties: false
+      properties:
+        certifiedAttributes:
+          $ref: "#/components/schemas/EServiceDescriptorAttributes"
+      required:
+        - certifiedAttributes
+    EServiceDescriptorDeclaredAttributesResponse:
+      type: object
+      additionalProperties: false
+      properties:
+        declaredAttributes:
+          $ref: "#/components/schemas/EServiceDescriptorAttributes"
+      required:
+        - declaredAttributes
+    EServiceDescriptorVerifiedAttributesResponse:
+      type: object
+      additionalProperties: false
+      properties:
+        verifiedAttributes:
+          $ref: "#/components/schemas/EServiceDescriptorAttributes"
+      required:
+        - verifiedAttributes
```

what do you think about this?
```suggestion
    EServiceDescriptorAttributes:
      type: object
      additionalProperties: false
      properties:
        attributes:
          $ref: "#/components/schemas/EServiceDescriptorAttributesGroups"
      required:
        - attributes
```

Renaming current `EServiceDescriptorAttributes` to `EServiceDescriptorAttributesGroups`